features/step_definitions/signup_definitions_step.rb

```diff
@@ -0,0 +1,19 @@
+Given("I am on the landing page") do
+  visit root_path 
+end
+
+When("I click on the {string} button") do |string|
+  click_on "Sign up"
+end
+
+Then("I fill in {string} with {string}") do |name, user_name|
+  fill_in(name, with: "Bob")
+end
+
+When("I click {string}") do |string|
+  click_on "Create"
```

Comment:
When you have updated the step on line 5 to use the variable you can drop this and modify the feature file to use the step on line 5